### PR TITLE
py-bluepyemodel: prepare for opensourcing

### PR DESCRIPTION
**Pull request by heerener**

### CODE REVIEW

**Comment by ilkilic** [comment]
I've removed the unused dependencies and tested it on BB5 using the specified command. @heerener please inform me if there are any further actions needed?

File: var/spack/repos/builtin/packages/py-bluepyemodel/package.py
Comment (AurelienJaquier) [inline var/spack/repos/builtin/packages/py-bluepyemodel/package.py]: why not use latest version, 0.0.43?

File: var/spack/repos/builtin/packages/py-bluepyemodel/package.py
Comment (AurelienJaquier) [inline var/spack/repos/builtin/packages/py-bluepyemodel/package.py]: same here, why not 0.0.43?

File: var/spack/repos/builtin/packages/py-bluepyemodel/package.py
Comment (AurelienJaquier) [inline var/spack/repos/builtin/packages/py-bluepyemodel/package.py:35]: also here the latest currentscape version on pypi is 0.1.1.dev12

Comment (matz-e) [reply var/spack/repos/builtin/packages/py-bluepyemodel/package.py:35]: Is there any release planned for that? These `devX` versions don't play nice with Spack's definition of versions.

File: var/spack/repos/builtin/packages/py-bluepyemodel/package.py
Comment (matz-e) [reply var/spack/repos/builtin/packages/py-bluepyemodel/package.py]: Does _not matter_ in the URI. This version string will get replaced by the stuff in `version(...)`

File: var/spack/repos/builtin/packages/py-bluepyemodel/package.py
Comment (AurelienJaquier) [reply var/spack/repos/builtin/packages/py-bluepyemodel/package.py:35]: I can make a release, yes. Should we have a spack package for currentscape as well or is having it on pypi enough?

Comment (heerener) [reply var/spack/repos/builtin/packages/py-bluepyemodel/package.py:35]: Wait, we're talking about https://pypi.org/project/currentscape/#history , right?
Then the latest version is 1.0.12

There's a spack recipe in the bluebrain repo that should probably be open-sourced as well (aka moved to the builtin repo and filed for upstream merging), just like several other dependencies (open-sourced, but spack recipe still in bluebrain). I'll move all of those to the builtin repo (and bother people with questions when I have them) on this PR.

Once this PR eventually gets merged in I can take care of upstreaming as well.

Comment (ilkilic) [reply var/spack/repos/builtin/packages/py-bluepyemodel/package.py:35]: @AurelienJaquier but if I put a colon after '0.0.11', won't it just install the newest version that fits? I'm using the same requirement as in our setup.py

Comment (heerener) [reply var/spack/repos/builtin/packages/py-bluepyemodel/package.py:35]: It should work like that, yes. I have an impression that upstream generally likes things to be as similar to setup.py as possible, so it might be better to leave it like this then.

Comment (AurelienJaquier) [reply var/spack/repos/builtin/packages/py-bluepyemodel/package.py:35]: ok! thanks for the explanations

**Review by matz-e — DISMISSED** [review]

**Comment by heerener** [comment]
I would hold off on the merging (and approving) until I get the dependencies sorted out as well

**Review by matz-e — COMMENTED** [review]
Rebase on `develop` to get the cellu lab?

**Review by matz-e — APPROVED** [review]

**Comment by ilkilic** [comment]
Thank you everyone, well done! @heerener is it safe to merge now?

**Comment by heerener** [comment]
> Thank you everyone, well done! @heerener is it safe to merge now?

It is!
I'll get to work on filing the upstream PR.

**Pull request merged**
